parcel/ParcelTendencies.jl

```diff
@@ -106,6 +107,39 @@ function immersion_freezing(params::P3_het, PSD, state)
     return max(FT(0), (Nᵢ_het - Nᵢ) / const_dt)
 end
 
+function immersion_freezing(params::Frostenberg_random, PSD, state)
+    FT = eltype(state)
+    (; ip, drawing_interval) = params
+    (; T, Nₗ, Nᵢ, t) = state
+    if mod(t, drawing_interval) == 0
+        μ = CMI_het.INP_concentration_mean(T)
+        INPC = exp(rand(Normal(μ, ip.σ)))
```

Comment:
Would it be possible to define the seed of the random number generator used here? We could even make it one of the parameters of the `_random` and `_stochastic` functions. This way our results will be reproducible, even when random

Reply:
I have some problems with passing the random seed as a parameter. I'm probably doing something wrong here, but when I try to use the random seed as a parameter to our functions, all of the results are equal. Instead, I tried to set different seeds before calling functions in the Example file. I'm attaching results here. It looks like the for the stochastic process, results average to the mean.
![frostenberg_immersion_freezing](https://github.com/CliMA/CloudMicrophysics.jl/assets/80834468/682995ed-f745-4b6f-bd6d-0582ddaad3da)

Reply:
I think you are right in how to use the seed and I think I found the typo in plotting. So maybe this will fix it?

Reply:
Thank you for noticing the typo! 
With the typo fixed, I think there is still something wrong here. I'm attaching two plots: in the first one, I'm averaging different realizations with selected seeds (as in this pull request). In the second one, I'm averaging different realizations, but without specifying the seed (deleting the line `RAND.seed!(trunc(Int, random_seed))`). There is something wrong with the first plot, in which I'm specifying the seed.
![frostenberg_immersion_freezing_seed](https://github.com/CliMA/CloudMicrophysics.jl/assets/80834468/88c908fe-7276-45cc-b112-e00c5712c39b)
![frostenberg_immersion_freezing](https://github.com/CliMA/CloudMicrophysics.jl/assets/80834468/e8c04048-033b-4959-9be7-c0211fefa993)